jobs/build-arch.Jenkinsfile

```
@@ -66,8 +66,12 @@ properties([
     durabilityHint('PERFORMANCE_OPTIMIZED')
 ])
 
+def cosa_img = params.COREOS_ASSEMBLER_IMAGE ?: pipeutils.get_cosa_img(pipecfg, params.STREAM)
```

would this work? 

```suggestion
def cosa_img = params.COREOS_ASSEMBLER_IMAGE ?: \
                   pipeutils.get_cosa_img(pipecfg, params.STREAM)
```

These lines are getting a little long

---

I'm not sure, but I reworked it some other other into two lines and added a clarifying comment.